reflections/type/type-test.js

```diff
@@ -85,7 +85,8 @@ QUnit.test("isMoreListLikeThanMapLike", function(){
 });
 
 QUnit.test("isObservableLike", function(){
-	ok(!typeReflections.isObservableLike({}), "Object");
+	ok(typeReflections.isObservableLike({}) === false, "Object");
+	ok(typeReflections.isObservableLike([]) === false, "Object");
```

Comment:
Should the message for this be `"Array"`?

Reply:
done